### PR TITLE
Add csv import feature for static deploy groups

### PR DESCRIPTION
**Pull request by ddurieux**

### CODE REVIEW

File: inc/deploygroup_staticdata.class.php [hunk 1]
```diff
@@ -208,6 +213,69 @@ static function duplicate($source_deploygroups_id, $target_deploygroups_id) {
...
+    */
+   static function csvimport_form(PluginFusioninventoryDeployGroup $item) {
+
+      echo "<form action='' method='post' enctype='multipart/form-data'>";
+
+      echo "<br>";
+      echo "<table class='tab_cadre_fixe' cellpadding='1' width='600'>";
+      echo "<tr>";
+      echo "<th>";
+      echo __('Import computers list from CSV file (first column must contain the computer ID)', 'fusioninventory')." :";
```
Comment (wawax) [inline inc/deploygroup_staticdata.class.php]: The local would be better like this : "Import a list of computers from a CSV file (the first column must contain the computer ID)".

File: inc/deploygroup_staticdata.class.php [hunk 2]
```diff
@@ -208,6 +213,69 @@ static function duplicate($source_deploygroups_id, $target_deploygroups_id) {
...
+      echo "</th>";
+      echo "</tr>";
+
+      echo "<tr class='tab_bg_1'>";
+      echo "<td align='center'>";
+      echo "<input type='hidden' name='groups_id' value='".$item->getID()."'/>";
```
Comment (wawax) [inline inc/deploygroup_staticdata.class.php]: Please use Html::hidden

File: inc/deploygroup_staticdata.class.php [hunk 3]
```diff
@@ -208,6 +213,69 @@ static function duplicate($source_deploygroups_id, $target_deploygroups_id) {
...
+      echo "<input type='file' name='importcsvfile' value=''/>";
+      echo "&nbsp;<input type='submit' value='".__('Import')."' class='submit'/>";
```
Comment (wawax) [inline inc/deploygroup_staticdata.class.php]: Please use Html::submit

File: inc/deploygroup_staticdata.class.php [hunk 4]
```diff
@@ -208,6 +213,69 @@ static function duplicate($source_deploygroups_id, $target_deploygroups_id) {
...
+      echo "</td>";
+      echo "</tr>";
+
+      echo "</table>";
+
+      Html::closeForm();
+      return TRUE;
```
Comment (wawax) [inline inc/deploygroup_staticdata.class.php]: Please use true and false instead of TRUE and FALSE :)

File: inc/deploygroup_staticdata.class.php [hunk 5]
```diff
@@ -208,6 +213,69 @@ static function duplicate($source_deploygroups_id, $target_deploygroups_id) {
...
+               }
+            }
+            Session::addMessageAfterRedirect(__('Computers imported successfully from CSV file', 'fusioninventory'), FALSE, INFO);
+            fclose($handle);
+         } else {
+            Session::addMessageAfterRedirect(__('Impossible to read the CSV file', 'fusioninventory'), FALSE, ERROR);
+            return false;
+         }
+      } else {
+         Session::addMessageAfterRedirect(__('CSV file not found', 'fusioninventory'), FALSE, ERROR);
```
Comment (wawax) [inline inc/deploygroup_staticdata.class.php]: You may avoid one local by using sprintf( __('%1$s %2$s'), "File not found", $files_data['importcsvfile']['tmp_name']);

**Comment by ddurieux** [comment]
Replaced by #2511

**Pull request closed without merging**